csharp/Squidex.ClientLibrary/Squidex.ClientLibrary.Tests/EnrichedEvents/EnrichedEventsTests.cs

```diff
@@ -106,6 +106,54 @@ public void Should_deserialize_EnrichedCommentEvent()
             Assert.Equal("testapp", contentEvent.App.Name);
             Assert.Equal("601c2cbafa4e669f214c0438", contentEvent.Actor.Id);
         }
+
+        public static string JsonEnrichedAssetEvent { get; } = @"{
```

Comment:
What I recently do is to create an anonymous object for that and then serialize it. Then you have less isues with formatting.

Reply:
I didn't know how to handle the `$type` property with an anonymous object. Copy and paste of a real json was faster ;)